2018/python/test_file_generator.py

```diff
@@ -23,6 +23,14 @@ def __init__(self, ts_schema, lar_schema, crosswalk_data):
 		#Loads the filepath configuration. 
 		with open('configurations/test_filepaths.yaml') as f:
 			filepaths = yaml.safe_load(f)
+
```

Comment:
Can you add a to the init doc string what the crosswalk data is?

Reply:
Sure! This change will be reflected in the next commit.